Gemfile.lock

```diff
@@ -45,10 +45,10 @@ GEM
       execjs
     bcrypt (3.1.11)
     bindex (0.5.0)
-    bootstrap (4.0.0.beta)
+    bootstrap (4.0.0.beta2.1)
```

Comment:
Please undo this line back to just `beta`. It seems small but unfortunately there are a bunch of breaking changes in this beta. There's an open PR for it. I just need to take the time to walk through the migration docs and make the changes in our repo. https://github.com/crimethinc/website/pull/508

Also, in the future on these kind of PRs, it's probably for the best if you scope your `bundle update` to the specific gems you're dealing with instead of the whole Gemfile. Especially since we've got @dependabot submitting version bump PRs all the time.

Reply:
fixed